operations/iscsi_sbps/Node_Personalization_Verification.md

```diff
@@ -0,0 +1,255 @@
+
+
+### Check for DNS SRV and A records
+
+    Check that all the DNS SRV and A records are created for all the intended worker nodes (worker nodes configured with iSCSI).
+    The nodes configured with iSCSI can be listed using:
+
+    ```bash
+    kubectl get nodes -l iscsi=sbps
+    ```
+
+    Example output:
+
+    ```text
+    NAME       STATUS   ROLES    AGE    VERSION
+    ncn-w001   Ready    <none>   7d2h   v1.32.5
+    ncn-w002   Ready    <none>   6d     v1.32.5
+    ncn-w003   Ready    <none>   7d1h   v1.32.5
+    ncn-w004   Ready    <none>   7d1h   v1.32.5
+    ```
+
+    Get the system domain name using:
+
+    ```bash
+    echo $SYSTEM_DOMAIN
+    ```
+    
+    Example output:
+
+   
+    ```text
+    drax.hpc.amslabs.hpecorp.net
+    ```
+   
+    Run the below command for both HSN and NMN interfaces using system domain obtained above to check DNS SRV records exist 
+    for the nodes configured with iSCSI:
+ 
+
+    ```bash
+    dig -t SRV +short _sbps-hsn._tcp.drax.hpc.amslabs.hpecorp.net _sbps-nmn._tcp.drax.hpc.amslabs.hpecorp.net
+    ```
+
+    Example output:
+
+    ```text
+    1 0 3260 iscsi-server-id-002.hsn.drax.hpc.amslabs.hpecorp.net.
+    1 0 3260 iscsi-server-id-003.hsn.drax.hpc.amslabs.hpecorp.net.
+    1 0 3260 iscsi-server-id-004.hsn.drax.hpc.amslabs.hpecorp.net.
+    1 0 3260 iscsi-server-id-001.hsn.drax.hpc.amslabs.hpecorp.net.
+    1 0 3260 iscsi-server-id-004.nmn.drax.hpc.amslabs.hpecorp.net.
+    1 0 3260 iscsi-server-id-002.nmn.drax.hpc.amslabs.hpecorp.net.
+    1 0 3260 iscsi-server-id-003.nmn.drax.hpc.amslabs.hpecorp.net.
+    1 0 3260 iscsi-server-id-001.nmn.drax.hpc.amslabs.hpecorp.net.
+    ```
+
+    Above example command output shows that DNS SRV records exist for all four worker nodes configured with iSCSI. Now run the below 
+    command to get DNS A records for each of the SRV record obtained:
+
+
+    ```bash
+    dig -t A +short iscsi-server-id-005.hsn.drax.hpc.amslabs.hpecorp.net. iscsi-server-id-002.hsn.drax.hpc.amslabs.hpecorp.net. iscsi-server-id-003.hsn.drax.hpc.amslabs.hpecorp.net. iscsi-server-id-004.hsn.drax.hpc.amslabs.hpecorp.net. iscsi-server-id-001.nmn.drax.hpc.amslabs.hpecorp.net. iscsi-server-id-002.nmn.drax.hpc.amslabs.hpecorp.net. iscsi-server-id-003.nmn.drax.hpc.amslabs.hpecorp.net. iscsi-server-id-004.nmn.drax.hpc.amslabs.hpecorp.net.
+    ```
+
+    Example output:
+
+    ```text
+    10.253.0.2
+    10.253.0.8
+    10.253.0.14
+    10.252.1.7
+    10.252.1.8
+    10.252.1.9
+    10.252.1.10
+    ```
+
+    The example output shows that there are DNS A records for each of the DNS SRV record.
+
+### Run readiness checks
+
+    In order to verify the overall readiness of the iSCSI targets before booting compute nodes or UANs, run GOSS tests to do
+    additional verification of the iSCSI targets.
+
+    See [GOSS tests for SBPS](https://github.com/Cray-HPE/sbps-marshal/blob/main/GOSS_tests_for_sbps.md) for more information.
```

Comment:
The content from that page needs to be included on this page, not kept in a different repo.